hoomd/hpmc/external/wall.py

```diff
@@ -95,8 +95,8 @@ class WallPotential(ExternalField):
 
     .. math::
 
-        U_{\mathrm{walls}} = \sum_{i=0}^{N_{\mathrm{particles}}}
-        \sum_{j=0}^{N_{\mathrm{walls}}} U_{i,j},
+        U_{\mathrm{walls}} = \sum_{i=0}^{N_{\mathrm{particles}-1}}
```

Comment:
We only support CPU execution here as well correct?

Reply:
Correct. I added a note.